app/services/notification_service.py

```
@@ -212,6 +272,170 @@ async def send_to_user(user_id: str) -> Dict[str, Any]:
+                f"카카오 Event API 배치 POST 요청 실패: status={post_payload.get('status')}, "
+                f"taskId={task_id}, batch_size={batch_size}"
+            )
+            return 0, batch_size
+
+        try:
+            task_response = await client.get(
+                NotificationService._kakao_task_url(str(task_id)),
+                headers=NotificationService._kakao_event_headers(),
+                timeout=settings.NOTIFICATION_TIMEOUT
+            )
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)